modules/gitops-release-notes-1-6-0.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assembly:
+//
+// * gitops/gitops-release-notes.adoc
+
+[id="gitops-release-notes-1-6-0_{context}"]
+= Release notes for {gitops-title} 1.6.0
+
+{gitops-title} 1.6.0 is now available on {product-title} 4.9, 4.10 and 4.11.
+
+[id="new-features-1-6-0_{context}"]
+== New features
+
+The current release adds the following improvements:
+
+* With this update, `ApplicationSet` is now available in GA and is merged to Argo CD core.
+
+* With this update, the parameters of `spec.sso` field that controlled the keycloak configurations are moved to `.spec.sso.keycloak`.
+The parameters of `.spec.dex` field are moved to `.spec.sso.dex`. Dex can now be enabled/disabled using `.spec.sso.provider`. The existing configuration methods will be supported alongside the new configuration methods until GitOps v1.9, when support for `.spec.dex`, `DISABLE_DEX` and `.spec.sso` fields for keycloak configuration will be removed from the operator link:https://issues.redhat.com/browse/GITOPS-1983[GITOPS-1983]
+
+* With this update, the Argo CD Notifications Controller is available as an optional workload that can be enabled/disabled through `.spec.notifications.enabled` parameter in the Argo CD custom resource. The Argo CD Notifications Controller is available as technical preview. link:https://issues.redhat.com/browse/GITOPS-1917[GITOPS-1917]
+
+* With this update, the default resource exclusions are added to the `csv` for Tekton pipeline runs and task runs. Argo CD, by default prunes these resources. These resource exclusions are added to the new Argo CD instances that are created from the *OpenShift console*. If the instances are created from the CLI the resources are not added. link:https://issues.redhat.com/browse/GITOPS-1876[GITOPS-1876]
+
+* With this update, you can set the value of the `resourceTrackingMethod` to select the tracking method used by Argo CD. link:https://issues.redhat.com/browse/GITOPS-1862[GITOPS-1862]
+
+* With this update, you can add entries to the `argocd-cm` configmap using the `extraConfig` field of Argo CD custom resource. The entries specified are reconciled to the live `config-cm` configmap without validations. link:https://issues.redhat.com/browse/GITOPS-1964[GITOPS-1964]
+
+* With this update, on {product-title} 4.11, the {gitops-title} **Environments Details** page in the {gitops-title} developer perspective shows history of the successful deployments of the application environments along with links to the revision for each deployment. link:https://issues.redhat.com/browse/GITOPS-1269[GITOPS-1269]
+
+* With this update, you can manage resources with Argo CD that are also being used as template resources or "source" by an Operator. link:https://issues.redhat.com/browse/GITOPS-982[GITOPS-982]
+
+* This release of {gitops-title} adds support for IBM Z and IBM Power on {product-title} 4.10.
```

Comment:
IBM P/Z on all supported OCP, right?  That is OCP 4.7-4.10.

Reply:
@SNiemann15 please suggest

Reply:
OK. I believe IBM P/Z are only tested on OCP 4.10 so the note is probably fine.

Reply:
Yes correct Gitops support for P/Z only on OCP 4.10